integrations/links_test.go

```diff
@@ -61,16 +61,6 @@ func TestRedirectsNoLogin(t *testing.T) {
 		resp := MakeRequest(t, req, http.StatusFound)
 		assert.EqualValues(t, path.Join(setting.AppSubURL, redirectLink), test.RedirectURL(resp))
 	}
-
-	var temporaryRedirects = map[string]string{
```

Comment:
But we can add a link test in line 92 of this file.

Reply:
Nope it doesn't redirect anymore